src/lib/ecl/attitude_fw/ecl_pitch_controller.cpp

```diff
@@ -96,15 +96,21 @@ float ECL_PitchController::control_attitude(const struct ECL_ControlData &ctl_da
 
 	/* input conditioning */
 	float airspeed = constrain_airspeed(ctl_data.airspeed, ctl_data.airspeed_min, ctl_data.airspeed_max);
-	/* calculate the offset in the rate resulting from rolling  */
-	//xxx needs explanation and conversion to body angular rates or should be removed
-	float turn_offset = fabsf((CONSTANTS_ONE_G / airspeed) *
-				  tanf(roll) * sinf(roll)) * _roll_ff;
```

Comment:
The roll feed forward is required to prevent the nose from dipping down during turns. Its not "right" but "works" very effectively. Do you have a proposal how to retain this compensation?